python/src/lib.rs

```diff
@@ -331,6 +335,29 @@ impl RawDeltaTable {
         Ok(serde_json::to_string(&metrics).unwrap())
     }
 
+
+    #[pyo3(signature = ())]
+    pub fn commit_optimize(
+        &mut self,
+    ) -> PyResult<String> {
+        let cmd = OptimizeBuilder::new(
+            self._table.log_store(),
+            self._table.snapshot().map_err(PythonError::from)?.clone(),
+        )
+        .with_max_concurrent_tasks(num_cpus::get());
+
+        let commit = match &self._commit {
+            Some(c) => c,
+            None => panic!("No commit found from previous optimize, cannot commit."),
+        };
+
+        rt()?.block_on(cmd.commit_writes(commit));
+
+        self._commit = None;
+
+        Ok(serde_json::to_string("test").unwrap())
```

Comment:
unexpected debug print?

Reply:
Yeah, was actually going to ask -- I'm not sure what a good return statement would be here.